{{cookiecutter.project_slug}}/config/settings/base.py

```diff
@@ -74,95 +73,91 @@ def env(name, default=NoDefaultValue, type_=str):
 
 # Quick-start development settings - unsuitable for production
 # See https://docs.djangoproject.com/en/1.11/howto/deployment/checklist/
-SECRET_KEY = env('DJANGO_SECRET_KEY')
-HASHID_FIELD_SALT = env('DJANGO_HASHID_SALT')
+SECRET_KEY = env("DJANGO_SECRET_KEY")
+HASHID_FIELD_SALT = env("DJANGO_HASHID_SALT")
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = env('DJANGO_DEBUG', default=False, type_=boolish)
+DEBUG = env("DJANGO_DEBUG", default=False, type_=boolish)
 
-MODE = env('DJANGO_MODE', default='dev' if DEBUG else 'prod')
+MODE = env("DJANGO_MODE", default="dev" if DEBUG else "prod")
 
 ALLOWED_HOSTS = [
-    '127.0.0.1',
-    '127.0.0.1:8000',
-    '127.0.0.1:8080',
-    'localhost',
-    'localhost:8000',
-    'localhost:8080',
+    "127.0.0.1",
+    "127.0.0.1:8000",
+    "127.0.0.1:8080",
+    "localhost",
+    "localhost:8000",
+    "localhost:8080",
 ] + [
     el.strip()
-    for el
-    in env('DJANGO_ALLOWED_HOSTS', default='', type_=lambda x: x.split(','))
+    for el in env("DJANGO_ALLOWED_HOSTS", default="", type_=lambda x: x.split(","))
     if el.strip()
 ]
 
 
 # Application definition
 
 INSTALLED_APPS = [
-    'django.contrib.admin',
-    'django.contrib.auth',
-    'django.contrib.sites',
-    'django.contrib.contenttypes',
-    'django.contrib.sessions',
-    'django.contrib.messages',
-    'channels',
-    'whitenoise.runserver_nostatic',
-    'django.contrib.staticfiles',
-    'django_rq',
-    'scheduler',
-    'allauth',
-    'allauth.account',
-    'allauth.socialaccount',
-    'rest_framework',
-    '{{cookiecutter.project_slug}}',
-    '{{cookiecutter.project_slug}}.multisalesforce',
-    '{{cookiecutter.project_slug}}.api',
-    'django_js_reverse',
+    "django.contrib.admin",
+    "django.contrib.auth",
+    "django.contrib.sites",
+    "django.contrib.contenttypes",
+    "django.contrib.sessions",
+    "django.contrib.messages",
+    "channels",
+    "whitenoise.runserver_nostatic",
+    "django.contrib.staticfiles",
+    "django_rq",
+    "scheduler",
+    "allauth",
+    "allauth.account",
+    "allauth.socialaccount",
+    "rest_framework",
+    "{{cookiecutter.project_slug}}",
+    "{{cookiecutter.project_slug}}.multisalesforce",
+    "{{cookiecutter.project_slug}}.api",
+    "django_js_reverse",
 ]
 
 MIDDLEWARE = [
-    'log_request_id.middleware.RequestIDMiddleware',
-    'django.middleware.security.SecurityMiddleware',
-    'whitenoise.middleware.WhiteNoiseMiddleware',
-    'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.common.CommonMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
-    'django.contrib.messages.middleware.MessageMiddleware',
-    'django.middleware.clickjacking.XFrameOptionsMiddleware',
+    "{{cookiecutter.project_slug}}.logging_middleware.LoggingMiddleware",
```

Comment:
@wlonk Is there a reason _not_ to backport this logging work?

Reply:
I don't think there's a reason not to, no.

Reply:
Cool; I pulled it in but commented out some potentially-helpful-but-currently-unused code. Please review the .py in this PR carefully 🙏